tests/MissingPhraseHttpException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Tnapf\Router;
+
+use Tnapf\Router\Exceptions\HttpException;
+
+class MissingPhraseHttpException extends HttpException
+{
+    public const CODE = 0;
+    public const PHRASE = "";
+    public const DESCRIPTION = "Indicates that the server encountered an unexpected
+condition that prevented it from fulfilling the request.";
```

Comment:
```suggestion
    public const DESCRIPTION
        = "Indicates that the server encountered an unexpected condition that prevented it from fulfilling the request.";
```
Something along these lines is what you should do to comply with that style guide

Reply:
Well issue is this will put \n into the string. Which is why I concat a bunch of together.